src/store/reducers/todosReducer/action-creators.ts

```diff
@@ -0,0 +1,105 @@
+import { AppDispatch } from '../..';
+import { IPureTodo, ITodo } from '../../../models/ITodo';
+import { WebService } from '../../../services/WebService';
+import {
+  TodosActionEnum,
+  SetTodosActionTodos,
+  AddTodoActionTodos,
+  SetErrorActionTodos,
+  DeleteTodoActionTodos,
+  UpdateTodoActionTodos,
+  UpdateTodoStatusActionTodos,
+  SetFilterActionTodos,
+} from './types';
+
+const webService = new WebService()
+
+export const TodosActionCreators = {
```

Comment:
Use uppercase naming if only it's a class or typescript entity.
Replace this object with a simple functions. Then export them 
```javascript
export {
  setTodos,
  fetchTodos,
  ...etc
}
```
You can import them as a default value
```javascript
import todosAction from './some_adress';
```

Reply:
done, noted